### PR TITLE
parsers(ccpuk): switch to TOML

### PR DESCRIPTION
**Pull request by abhidg**

### CODE REVIEW

File: isaric/parsers/isaric-ccpuk.toml [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+
+[subject.pregnancy]
+field = "pregyn_rptestcd"
+ref = "Y/N/NK"
+description = "Pregnant ?"
+
+[subject.pregnancy_date_of_delivery]
+comment = "considers delivery before and after admission"
+combinedType = "firstNonNull"
+fields = [
```
Comment (ekamau) [inline isaric/parsers/isaric-ccpuk.toml:96]: This part looks different from other combinedType entries ... no square brackets [[ ]]. or it doesn't matter?

File: isaric/parsers/isaric-ccpuk.toml [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+country_iso3 = "GBR"
+
+## SUBJECT
+
+[subject]
+study_id = "isaric-ccpuk"
+country_iso3 = "GBR"
+pathogen = "COVID-19"
+
+[visit]
```
Comment (ekamau) [inline isaric/parsers/isaric-ccpuk.toml:26]: Should these two lines be here? are they misplaced?

**Review by ekamau — APPROVED** [review]
TOML looks more compact than json, less bulky, perhaps easier to read.

File: isaric/parsers/isaric-ccpuk.toml (hunk 2 above)
Comment (pipliggins) [reply isaric/parsers/isaric-ccpuk.toml:26]: I agree, I'd put these with the rest of the visit table

File: isaric/parsers/isaric-ccpuk.toml (hunk 1 above)
Comment (abhidg) [reply isaric/parsers/isaric-ccpuk.toml:96]: Double square brackets are list of maps (array of tables, https://toml.io/en/v1.0.0#array-of-tables), so these are identical:

```toml
fields = [
  { field = "hello" },
  { field = "world" }
]
```
and
```toml
[[fields]]
field = "hello"

[[fields]]
field = "world"
```

File: isaric/parsers/isaric-ccpuk.toml (hunk 2 above)
Comment (abhidg) [reply isaric/parsers/isaric-ccpuk.toml:26]: Thanks, fixed!

**Pull request merged**